app/views/shared/_footer.html.erb

```diff
@@ -13,17 +13,13 @@
       </div>
     </div>
     <div class="black row links">
-      <div class="col-xs-3">
-        <a class="white" href="https://oregondigital.org/contact"> Contact Us </a>
-      </div>
-      <div class="col-xs-3">
-        <a class="white" href="https://oregondigital.org/copyright"> Copyright </a>
-      </div>
-      <div class="col-xs-3">
-        <%= link_to "Help", help_path  %>
-      </div>
-      <div class="col-xs-3">
-        <%= link_to "About Us", about_index_path %>
+      <div class="col-xs-12 col-md-offset-4 col-md-4">
```

Comment:
So rather than adding classes everywhere I'd prefer just having specific CSS that references an appropriate ID and describes its children.